SpatialConnect/SCConfigService.m

```diff
@@ -112,10 +113,15 @@ - (void)loadConfig:(SCConfig *)c {
   if (c.remote) {
     if ([c.remote.auth isEqualToString:@"no-auth"]) {
       [sc connectAuth:[[SCNoAuthMethod alloc] init]];
+    } else if ([c.remote.auth isEqualToString:@"exchange"]) {
+      [sc connectAuth:[[SCExchangeAuthMethod alloc] initWithDictionary:@{
+            @"server_url" : c.remote.httpUri,
+            @"client_id" : c.remote.clientId
```

Comment:
We shouldn't need to look for the clientId anymore so we should remove this

Reply:
We can fix it in another PR...for now, keeping the client_id as a required key is fine